Lib/test/_test_multiprocessing.py

```diff
@@ -5536,6 +5536,37 @@ def test_too_long_name_resource(self):
         with self.assertRaises(ValueError):
             resource_tracker.register(too_long_name_resource, rtype)
 
+    def _test_resource_tracker_leak_resources(self, context, delete_queue):
+        from multiprocessing.resource_tracker import _resource_tracker
+        _resource_tracker.ensure_running()
+        self.assertTrue(_resource_tracker._check_alive())
+
+        # Reset exit code value
+        _resource_tracker._exitcode = None
+        exit_code_assert = self.assertNotEqual
```

Comment:
This is a bit confusing. Also, it will match any non-zero exit code which might not be desired (other problems may return some code not in `{0, 1}`). How about simply `expected_exit_code = 0 if delete_queue else 1`?

Reply:
Hey, 
I fixed the test and fixed the code too. 
Since I saved the exit status (which was 256 when the exit code was 1), 
so now the code saves the exit code and verifies it to `0/1`
Thanks